### ISSUE TITLE
Rebrand - FF Setup Screens

### ISSUE THREAD

**Issue by joepavitt** [flowforge/flowforge#500]
Update branding on the "setup" workflow for when a user first runs and configures FlowForge.

**Comment by joepavitt** [comment]
A lot of this uses `FormRow`, which I am leaving in for now. styling-wise it works fine, although is inconsistent with the Login/Signup screens I've already done. 

I had avoided using `FormRow` as any changes there would echo across the entire platform, which prior to #501, we don't want to do.

It means some of the setup workflow styles slightly differently to where we'd like it, but once #501 is complete, we can come back and check to make sure this is unaffected here.
